src/status_im2/contexts/communities/actions/addresses_for_permissions/view.cljs

```diff
@@ -0,0 +1,54 @@
+(ns status-im2.contexts.communities.actions.addresses-for-permissions.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im2.contexts.communities.actions.addresses-for-permissions.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- account-item
+  [item]
+  [rn/view
+   {:style style/account-item-container}
+   [quo/account-avatar (assoc item :size 32)]
+   [rn/view
+    [quo/text
+     {:size   :paragraph-1
+      :weight :semi-bold} (:name item)]
+    [quo/address-text
+     {:address (:address item)
+      :format  :short}]]])
+
+(defn view
+  []
+  (let [{id :community-id}          (rf/sub [:get-screen-params])
+        {:keys [name color images]} (rf/sub [:communities/community id])
+        accounts                    (->> (rf/sub [:wallet])
+                                         :accounts
+                                         vals
+                                         (map #(assoc % :customization-color (:color %))))]
+    [rn/safe-area-view {:style style/container}
+
+     [quo/drawer-top
```

Comment:
hm, I was not aware about this. Think I will also implement on mine.

Can you see the tag background color? Mine is transparent.

Also, what If we include the info button as part of this PR?

Reply:
> hm, I was not aware about this. Think I will also implement on mine.

Sure

> Can you see the tag background color? Mine is transparent.

Background is there
<img width="340" alt="Screenshot 2023-12-15 at 09 59 44" src="https://github.com/status-im/status-mobile/assets/19279756/366588aa-0fcb-483e-aa1c-fc776a394f0c">


> Also, what If we include the info button as part of this PR?

Done, the quo component requires some updates to fix the button background.